### PR TITLE
chore: remove beta info

### LINKED ISSUE #898: [chore]: Remove beta warning
Before GA we should remove the beta log warning

### PR DESCRIPTION
**Pull request by ascorbic**
Removes the beta warning and updates the README to reflect the fact this is GA. Includes updates to migration and install guide.

Do not merge til 4.0.0 is released!

Fixes #898

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,36 +16,46 @@
...
+
+Change the `publish` directory to `.next`:
+
+```toml
+[build]
+publish = ".next"
+```
+
+If you previously set a custom `distDir` in your `next.config.js`, or set `node_bundler = "esbuild"` or
+`external_node_modules` in your `netlify.toml` these are no longer needed and can be removed.
```
Comment (KyleBlankRollins) [inline README.md:53]: [sand] Structure edit to improve readability.

```suggestion
If you previously set these values, they're no longer needed and can be removed:
- `distDir` in your `next.config.js`
- `node_bundler = "esbuild"` in `netlify.toml`
- `external_node_modules` in `netlify.toml`
```

File: README.md [hunk 2]
```diff
@@ -19,36 +16,46 @@
 Version 4 is a complete rewrite of the Essential Next.js plugin. For full details of everything that's new, check out
 [the v4 release notes](https://github.com/netlify/netlify-plugin-nextjs/blob/main/docs/release-notes/v4.md)
 
-## Installing the beta
+## Installing the plugin
+
+The plugin should be installed automatically for new Next.js sites on Netlify. You can also install it manually like
```
Comment (KyleBlankRollins) [inline README.md:21]: [sand] Minor suggestion for concision.
```suggestion
The plugin installs automatically for new Next.js sites on Netlify. You can also install it manually like
```

**Review by KyleBlankRollins — COMMENTED** [review]
I have some minor suggestions for the content in this PR. I know this is still in draft, but wanted to get a head of review work for the beta. Happy to take another look if needed.

**Comment by ascorbic** [comment]
@KyleBlankRollins Thanks!

**Review by tiffanosaurus — APPROVED** [review]
🚀🚀

**Pull request merged**